compiler/src/iree/compiler/Codegen/Dialect/GPU/Transforms/TilingInterfaceImpl.cpp

```diff
@@ -0,0 +1,161 @@
+// Copyright 2024 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUDialect.h"
+#include "iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUOps.h"
+#include "mlir/Dialect/Bufferization/IR/BufferizableOpInterface.h"
+#include "mlir/Dialect/GPU/IR/GPUDialect.h"
+#include "mlir/Dialect/Tensor/IR/Tensor.h"
+#include "mlir/IR/BuiltinTypes.h"
+#include "mlir/IR/OpDefinition.h"
+
+namespace mlir::iree_compiler::IREE::GPU {
+
+//===----------------------------------------------------------------------===//
+// MultiMmaOp
+//===----------------------------------------------------------------------===//
+
+SmallVector<utils::IteratorType> MultiMmaOp::getLoopIteratorTypes() {
+  return getIteratorTypesArray();
+}
+
+static int64_t getResultIndex(AffineMap map, AffineExpr targetExpr) {
+  for (int64_t i = 0, e = map.getNumResults(); i < e; ++i)
+    if (targetExpr == map.getResult(i))
+      return i;
+  return -1;
+}
```

Comment:
You can use `AffineMap::getResultPosition(targetExpr)`
https://github.com/llvm/llvm-project/blob/c31ac810910ac87531de636ea508abec6e29e8ff/mlir/lib/IR/AffineMap.cpp#L419-L429